.github/workflows/build-for-release.yml

```diff
@@ -78,30 +78,28 @@ jobs:
       uses: actions/checkout@v3
       with:
         fetch-depth: 0  # IMPORTANT! to get all the branches
-    - name: Ensure that the specified commit exists in the latest release branch
-      env:
-        COMMIT: ${{ inputs.commit }}
+        sparse-checkout: release
```

Comment:
Just to keep this in mind:
> THIS COMMAND IS EXPERIMENTAL. ITS BEHAVIOR, AND THE BEHAVIOR OF OTHER COMMANDS IN THE PRESENCE OF SPARSE-CHECKOUTS, WILL LIKELY CHANGE IN THE FUTURE.[^1]

[^1]: https://www.git-scm.com/docs/git-sparse-checkout#_description

Reply:
oh interesting, I didn't realize that. it makes checkout go MUCH faster. like 30s --> 5s